sdk/avs/Azure.ResourceManager.Avs/src/Customized/WorkloadNetworksRestOperations.cs

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager.Avs.Models;
+
+namespace Azure.ResourceManager.Avs
+{
+    internal partial class WorkloadNetworksRestOperations
```

Comment:
I think we do not need this class at all - if the new operations are missing rest operation to call, they could call the new ones with name parameter

Reply:
WorkloadNetwordsRestOperations is used by WorkloadNetworkCollections which is used once in AvsPrivateCloudResource in the GetWorkloadNetworks() method discussed above

Reply:
So initially I think those methods that are calling this old version of Get here (which requires a name), could be replaced by the new get methods without a `name` parameter.
But then I saw we have a `Exists` method which accepts a name of any value, therefore removing this is not reasonable any more.
So let us just keep this. Sorry for the inconvenience.